src/applications/caregivers/config/form.js

```diff
@@ -192,7 +192,6 @@ const formConfig = {
     },
     secondaryCaregiversTwoChapter: {
       title: secondaryTwoChapterTitle,
-      depends: formData => hasSecondaryCaregiverTwo(formData),
```

Comment:
Was this intentional?

Reply:
Sure is!! `Depends` doesn't actually work on chapters. It only works on pages and is duplicated with the pages in this chapter. So just getting rid of an unused line.